examples/neighbors/plot_lmnn_dim_reduction.py

```diff
@@ -0,0 +1,76 @@
+"""
+======================================================
+Large Margin Nearest Neighbor Dimensionality Reduction
+======================================================
+
+Sample usage of Large Margin Nearest Neighbor for dimensionality reduction.
```

Comment:
Can say a tiny bit more here, so that a user who Googles and lands on this page understands better why and what.

Reply:
Done.